app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -98,6 +129,14 @@ def forward_to_mas?
       Flipper.enabled?(:rrd_mas_disability_tracking)
   end
 
+  def new_pact_related_disability?
```

Comment:
Renaming to capture that the disability must be in the list of codes that RRD can handle, i.e., not _all_ new pact disabilities.
```suggestion
  def new_pact_rrd_related_disability?
```